src/main/java/com/untamedears/humbug/Humbug.java

```diff
@@ -2383,53 +2384,48 @@ public void run() {
   public final static int pearlId = 368;
   public final static MinecraftKey pearlKey = new MinecraftKey("ender_pearl");
 
-  @SuppressWarnings({ "rawtypes", "unchecked" })
+  @SuppressWarnings("unchecked")
   @BahHumbug(opt="ender_pearl_gravity", type=OptType.Double, def="0.060000")
-  private void hookEnderPearls() {
+  public void hookEnderPearls() {
```

Comment:
Already changed to 1.10.2.

We have just Bastion which has dependency on Humbug.
Also I've marked some field in Humbug as private and exposed method instead of this, so update is needed anyway.